html/pfappserver/root/static.alt/src/globals/configuration/pfConfigurationSwitchGroups.js

```diff
@@ -484,9 +507,10 @@ export const pfConfigurationSwitchGroupViewFields = (context = {}) => {
     {
       tab: i18n.t('Roles'),
       fields: [
-        { label: i18n.t('Role mapping by VLAN ID'), labelSize: 'lg' },
+        { label: i18n.t('Role mapping by VLAN ID'), labelSize: 'lg', if: switchTypeSupport(['RadiusDynamicVlanAssignment']) },
```

Comment:
Reorder so the if statement is first so conditions are more visible. Apply to all additions below and in other files.